spec/lib/myfinance/resources/person_spec.rb

```diff
@@ -0,0 +1,226 @@
+        expect(subject.created_at).to eq(DateTime.parse("2015-11-12T14:34:37-02:00"))
+        expect(subject.customer).to eq(true)
+        expect(subject.email).to eq("")
+        expect(subject.excel_import_id).to be_nil
+        expect(subject.febraban_favored_code).to be_nil
+        expect(subject.febraban_name).to be_nil
+        expect(subject.federation_subscription_number).to eq("42.308.611/0001-41")
+        expect(subject.federation_subscription_number_only_numbers).to eq("42308611000141")
+        expect(subject.federation_subscription_type_id).to be_nil
+        expect(subject.force_destroy).to eq(false)
+        expect(subject.guid).to be_nil
+        expect(subject.imported_from_sync).to eq(false)
+        expect(subject.interested_users_ids).to eq([3691])
+        expect(subject.modified_by_sync).to eq(false)
+        expect(subject.name).to eq("Myfreecomm")
+        expect(subject.neighborhood).to eq("")
+        expect(subject.note).to eq("")
+        expect(subject.person_type).to eq("JuridicalPerson")
+        expect(subject.phone).to eq("")
+        expect(subject.site).to eq("")
+        expect(subject.state).to eq("")
+        expect(subject.supplier).to eq(true)
+        expect(subject.updated_at).to eq(DateTime.parse("2015-11-12T14:34:37-02:00"))
+        expect(subject.use_count).to eq(0)
+        expect(subject.zip_code).to eq("")
+      end
+    end
+
+    context "when not found" do
+      let(:client) { Myfinance.client("") }
+      subject { client.people.find(199424) }
+
+      it "raises NotFound" do
+        expect { subject }.to raise_error(Myfinance::RequestError)
+      end
+    end
+  end
+
+  describe "#create", vcr: true do
+    let(:params) do
+      {
+        person: {
+          address: "Rua Tal",
+          address_number: "42",
+          city: "Rio de Janeiro",
+          complement: "sala 1234",
+          country: "Brasil",
+          customer: true,
+          email: "cliente@fulano.com",
+          federation_subscription_number: "27.206.831/0001-70",
+          name: "Fulano de Tal",
+          neighborhood: "Centro",
+          note: "Notas sobre este cliente...",
+          person_type: "JuridicalPerson",
+          phone: "(21) 5555-1234",
+          site: "http://www.fulano.com",
+          state: "RJ",
+          supplier: false,
+          zip_code: "22290-080"
+        }
+      }
+    end
+
+    context "when success" do
+      subject { client.people.create(params) }
+
+      it "returns a created person" do
+        expect(subject.class).to eq(Myfinance::Entities::Person)
+        expect(subject.id).to eq(199425)
+        expect(subject.address).to eq("Rua Tal")
+        expect(subject.address_number).to eq("42")
+        expect(subject.city).to eq("Rio de Janeiro")
+        expect(subject.complement).to eq("sala 1234")
+        expect(subject.country).to eq("Brasil")
+        expect(subject.created_at).to eq(DateTime.parse("2015-11-12T16:12:55-02:00"))
```

Comment:
Do not use `DateTime.parse` without zone. Use one of `Time.zone.parse`, `DateTime.parse.current`, `DateTime.parse.in_time_zone`, `DateTime.parse.utc`, `DateTime.parse.getlocal`, `DateTime.parse.iso8601`, `DateTime.parse.jisx0301`, `DateTime.parse.rfc3339`, `DateTime.parse.to_i`, `DateTime.parse.to_f` instead.